api/src/main/java/org/apache/cloudstack/api/command/admin/cluster/AddClusterCmd.java

```diff
@@ -204,6 +210,10 @@ public ApiCommandResourceType getApiResourceType() {
         return ApiCommandResourceType.Cluster;
     }
 
+    public CPU.CPUArch getArch() {
+        return CPU.CPUArch.fromType(arch);
```

Comment:
check if this is Ok for null arch (as it is optional)

Reply:
Hi @sureshanaparti it will fallback to the default arch (x86_64) as in DB we won't have null values for clusters

Reply:
The same applies for the other getArch() getters